lib/drives/pages/drive_detail_page.dart

```diff
@@ -328,17 +255,18 @@ class _DriveDetailPageState extends State<DriveDetailPage> {
                     child: Align(
                       alignment: Alignment.centerLeft,
                       child: Row(
-                          mainAxisSize: MainAxisSize.min,
-                          children: action.seats <= 2
-                              ? List.generate(action.seats, (index) => icon)
-                              : [
-                                  icon,
-                                  const SizedBox(width: 2),
-                                  Text("x${action.seats}"),
-                                ]),
+                        mainAxisSize: MainAxisSize.min,
+                        children: action.seats <= 2
+                            ? List.generate(action.seats, (index) => icon)
+                            : [
+                                icon,
+                                const SizedBox(width: 2),
+                                Text("x${action.seats}"),
+                              ],
+                      ),
                     ),
                   ),
-                  profileRow(profile),
+                  ProfileRow(profile, size: 15),
                   const Expanded(
```

Comment:
can we move this icon in ProfileRow if we have the attribute showChatButton there

Reply:
Ah sorry that attribute is legacy. We originally wanted to do what you are saying, but we decided against it because there could be all kinds of icons on the right side. This widget here should really just show the avatar and the name.

I'm gonna remove the attribute and maybe rename ProfileRow to ProfileWidget or something.